Mathlib/Data/Polynomial/Reverse.lean

```diff
@@ -344,6 +347,27 @@ theorem coeff_one_reverse (f : R[X]) : coeff (reverse f) 1 = nextCoeff f := by
     exact Nat.succ_le_iff.2 (pos_iff_ne_zero.2 hf)
 #align polynomial.coeff_one_reverse Polynomial.coeff_one_reverse
 
+@[simp] lemma reverse_C {t : R} :
+    reverse (C t) = C t := by
+  simp [reverse]
+
+@[simp] lemma reverse_mul_X {p : R[X]} : reverse (p * X) = reverse p := by
+  nontriviality R
+  rcases eq_or_ne p 0 with rfl | hp
+  · simp
+  · simp [reverse, hp]
+
+@[simp] lemma reverse_X_mul {p : R[X]} : reverse (X * p) = reverse p := by
+  rw [commute_X p, reverse_mul_X]
```

Comment:
Do you think it's worth having `X_pow` versions of these too?

Reply:
Sure, done.